### PR TITLE
[DOCS] Add Zabbix monitoring template to community monitoring integrations

### PR DESCRIPTION
**Pull request by RogerTheUnicornHive**
Added community monitoring integration by Zabbix.

### CODE REVIEW

**Comment by RogerTheUnicornHive** [comment]
The CLA has been signed. :)

File: docs/plugins/integrations.asciidoc [hunk 1]
```diff
@@ -188,7 +188,8 @@ releases 2.0 and later do not support rivers.
 * http://sematext.com/spm/index.html[SPM for Elasticsearch]:
   Performance monitoring with live charts showing cluster and node stats, integrated
   alerts, email reports, etc.
-
+* https://www.zabbix.com/integrations/elasticsearch[Zabbix monitoring template]:
+  Monitor the performance and status of your elastic search nodes and cluster with Zabbix, and receive events information.
```
Comment (jrodewig) [inline docs/plugins/integrations.asciidoc:192]:
```suggestion
* https://www.zabbix.com/integrations/elasticsearch[Zabbix monitoring template]:
  Monitor the performance and status of your {es} nodes and cluster with Zabbix,
  and receive events information.
```

**Comment by jrodewig** [comment]
@elasticmachine test this please

File: docs/plugins/integrations.asciidoc [hunk 2]
```diff
@@ -188,7 +188,9 @@ releases 2.0 and later do not support rivers.
 * http://sematext.com/spm/index.html[SPM for Elasticsearch]:
   Performance monitoring with live charts showing cluster and node stats, integrated
   alerts, email reports, etc.
-
+* https://www.zabbix.com/integrations/elasticsearch[Zabbix monitoring template]:
+  Monitor the performance and status of your {es} nodes and cluster with Zabbix,
+  and receive events information.
```
Comment (jrodewig) [inline docs/plugins/integrations.asciidoc:193]:
```suggestion
  Monitor the performance and status of your {es} nodes and cluster with Zabbix
  and receive events information.
```

**Review by jrodewig — APPROVED** [review]
LGTM. Thanks for your contribution @RogerTheUnicornHive.

**Comment by jrodewig** [comment]
### Backport commits
master 8c01f32 
7.x 0cb5404
7.7 f67172f
7.6 c49b8a1

**Pull request merged**